msu/systems/persistent_data/persistent_data_system.nut

```diff
@@ -0,0 +1,109 @@
+::MSU.Class.PersistentDataSystem <- class extends ::MSU.Class.System
+{
+	Mods = null;
+	ModConfigPath = "mod_config/";
+
+	constructor()
+	{
+		base.constructor(::MSU.SystemID.PersistentData);
+		this.Mods = {};
+	}
+
+	function registerMod( _mod )
+	{
+		base.registerMod(_mod);
+
+		_mod.PersistentData = ::MSU.Class.PersistentDataModAddon(_mod);
+		this.addMod(_mod.getID());
```

Comment:
at some point I feel like we should use mod addons for storage and make these tables store mods or something, but I know keybinds has a problem with this. Idk, just bothers me.

Reply:
Yeah, it's all a bit out of whack.

Reply:
For 1.1